src/instructlab/utils.py

```diff
@@ -350,6 +350,24 @@ def retrieve(uri: str) -> Resource:
     return errors
 
 
+def validate_json_file(json_filepath: Path) -> bool:
```

Comment:
It seemed like a general utility function that we can write once and reuse wherever we need to validate json files, instead of duplicating that check everywhere 
I'm happy to remove it if you feel strongly though

Reply:
If the function was doing something else than just validating the file, like checking for fields etc I would say it makes sense to have it. But right now, I'm sure it's helping much but maybe others think differently...

Reply:
I strongly advise against unstructured JSON or unstructured data in general. It's a source of bugs and pain. All JSON or YAML documents have to be defined by a schema and validated..

Reply:
@tiran sorry, not quite sure I follow what you're suggesting here
If you're suggesting that we need more strict JSON validation, maybe this function can be expanded to do schema validation in a future PR (and maybe there is value in keeping it here then?)

Reply:
I think for the scope of this PR we remove safetensor model json file validation right now and make it a TODO along with verifying .safetensors files are valid as well.

Reply:
Discussed offline with @tiran and we would like to remove this function as suggested previously and simply do try-except in place. Thanks!

Reply:
removed, thanks!